client/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from buy_together_app.models import User
+
+
+class Client(User):
+    area = models.CharField(max_length=32,
+                            validators=[MinLengthValidator(1)],
+                            help_text="area")
+
+    @staticmethod
+    def get_all_clients():
+        """Fetches list of all Clients.
+        Args:
+            None.
+        Returns:
+            returns a queryset containing all the Clients.
+        """
+        return Client.objects.all()
+
+    @staticmethod
```

Comment:
Why this function a static function and not regular object function,if you have the object client already why not just call save on the object self.